tokens.js

```diff
@@ -867,13 +867,18 @@ const getAllWithdrawsDeposits = contractName => async chainName => {
     polygonChainName,
   } = getChainNames(chainName);
 
+  // Polygon getPastEvent calls must be rated.
+  let getEventsHandler = chainName = 'polygon'
```

Comment:
Could be const.

Reply:
Good catch, a relic of converting an if/else to a ternary statement.